tests/integration/targets/aci_interface_policy_leaf_fc_policy_group/tasks/main.yml

```
@@ -0,0 +1,232 @@
+      attached_entity_profile: policy_group_aep
+      state: present
+    check_mode: yes
+    register: intf_policy_leaf_access_fc_pc_polgrp_check_mode_present
+
+  - name: Adding a interface policy Leaf Access FC PC policy group - creation works
+    cisco.aci.aci_interface_policy_leaf_fc_policy_group:
+      <<: *fc_pc_present
+    register: intf_policy_leaf_access_fc_pc_polgrp_present
+
+  - name: Adding a interface policy Leaf Access FC PC policy group  - idempotency works
+    cisco.aci.aci_interface_policy_leaf_fc_policy_group:
+      <<: *fc_pc_present
+    register: intf_policy_leaf_access_fc_pc_polgrp_idempotent
+
+  # UPDATE Leaf Access FC PC Policy Group
+  - name: Adding a interface policy Leaf Access FC PC policy group description - update works
+    cisco.aci.aci_interface_policy_leaf_fc_policy_group:
+      <<: *fc_pc_present
+      description: policygroup description
+    register: intf_policy_leaf_access_fc_pc_polgrp_update
+
+  - name: Verify present assertions for FC PC Policy Group
+    assert:
+      that:
+      - intf_policy_leaf_access_fc_pc_polgrp_check_mode_present is changed
+      - intf_policy_leaf_access_fc_pc_polgrp_present is changed
+      - intf_policy_leaf_access_fc_pc_polgrp_present.previous == []
+      - intf_policy_leaf_access_fc_pc_polgrp_present.current[0].infraFcAccBndlGrp.attributes.name == 'fc_pc_test'
+      - intf_policy_leaf_access_fc_pc_polgrp_present.current[0].infraFcAccBndlGrp.children.0.infraRsFcLagPol.attributes.tnLacpLagPolName == 'port_channel_policy_test'
+      - intf_policy_leaf_access_fc_pc_polgrp_present.current[0].infraFcAccBndlGrp.children.1.infraRsFcAttEntP.attributes.tDn == 'uni/infra/attentp-policy_group_aep'
+      - intf_policy_leaf_access_fc_pc_polgrp_present.current[0].infraFcAccBndlGrp.children.2.infraRsFcL2IfPol.attributes.tnFcIfPolName == 'fibre_channel_interface_policy_test'
+      - intf_policy_leaf_access_fc_pc_polgrp_present.current[0].infraFcAccBndlGrp.attributes.annotation == 'orchestrator:ansible'
+      - intf_policy_leaf_access_fc_pc_polgrp_idempotent is not changed
+      - intf_policy_leaf_access_fc_pc_polgrp_idempotent.sent == {}
+      - intf_policy_leaf_access_fc_pc_polgrp_update is changed
+      - intf_policy_leaf_access_fc_pc_polgrp_update.current[0].infraFcAccBndlGrp.attributes.descr  == 'policygroup description'
+
+
+  # QUERY Leaf Access FC Port Policy Group
+  - name: Query all interface policy Leaf Access FC Port policy groups
+    cisco.aci.aci_interface_policy_leaf_fc_policy_group:
+      <<: *aci_info 
+      lag_type: port
+      state: query
+    register: query_all_leaf_access_fc_port_policy_groups
+
+  - name: Query interface policy Leaf Access FC Port policy group
+    cisco.aci.aci_interface_policy_leaf_fc_policy_group:
+      <<: *aci_info 
+      policy_group: fc_port_test
+      lag_type: port
+      state: query
+    register: query_leaf_access_fc_port_policy_group
+
+  - name: Verify query assertions
```

Use different task name
```suggestion
  - name: Verify query Leaf Access FC Port Policy Group
```